### PR TITLE
fixes issue #293

### PR DESCRIPTION
**Pull request by chrissparksnj**
added new methods to the testing module, along with a file full of unparsed html.

### CODE REVIEW

File: output.txt [hunk 1]
```diff
@@ -0,0 +1,864 @@
+
```
Comment (L1ghtn1ng) [inline output.txt]: This file needed? if not please remove

File: tests/discovery/test_linkedin_links.py [hunk 2]
```diff
@@ -12,5 +13,16 @@ def test_get_links(self):
         links = search.get_links()
         assert list(links)
 
+    def test_links_linkedin(self):
+        f = open("test_linkedin_links.txt").read()
```
Comment (L1ghtn1ng) [inline tests/discovery/test_linkedin_links.py]: Can you update single char variable names please say what they are doing to make it better for correctiveness and code understanding

File: tests/discovery/test_linkedin_links.py [hunk 3]
```diff
@@ -12,5 +13,16 @@ def test_get_links(self):
...
+        reg_links = re.compile(r"url=https:\/\/www\.linkedin.com(.*?)&")
+        temp = reg_links.findall(f)
+        resul = []
+        for x in temp:
+            y = x.replace("url=", "")
```
Comment (L1ghtn1ng) [inline tests/discovery/test_linkedin_links.py]: same as above

File: tests/discovery/test_linkedin_links.py [hunk 4]
```diff
@@ -12,5 +13,16 @@ def test_get_links(self):
         links = search.get_links()
         assert list(links)
 
+    def test_links_linkedin(self):
+        f = open("test_linkedin_links.txt").read()
+        reg_links = re.compile(r"url=https:\/\/www\.linkedin.com(.*?)&")
+        temp = reg_links.findall(f)
+        resul = []
+        for x in temp:
```
Comment (L1ghtn1ng) [inline tests/discovery/test_linkedin_links.py]: same as above

File: theHarvester/discovery/linkedinsearch.py [hunk 5]
```diff
@@ -35,6 +35,7 @@ def get_people(self):
         return rawres.people_linkedin()
 
     def get_links(self):
+        print(self.totalresults)
```
Comment (L1ghtn1ng) [inline theHarvester/discovery/linkedinsearch.py]: Is this needed or was it for testing?

**Comment by L1ghtn1ng** [comment]
@chrissparksnj  also travis is failing on
 ```
self = <tests.discovery.test_linkedin_links.TestGetLinks object at 0x7f13f6d3aa58>

    def test_links_linkedin(self):

       f = open("test_linkedin_links.txt").read()

E       FileNotFoundError: [Errno 2] No such file or directory: 'test_linkedin_links.txt'

tests/discovery/test_linkedin_links.py:17: FileNotFoundError

========================= 2 failed, 9 passed in 1.77s ==========================

The command "pytest" exited with 1.
```

File: theHarvester/discovery/linkedinsearch.py (hunk 5 above)
Comment (chrissparksnj) [reply theHarvester/discovery/linkedinsearch.py]: just testing. just reverted it.. working on other code now

**Comment by L1ghtn1ng** [comment]
@chrissparksnj  Please fix https://travis-ci.com/laramies/theHarvester/jobs/232109259

File: tests/discovery/test_linkedin_links.py [hunk 6]
```diff
@@ -17,12 +17,12 @@ def test_get_links(self):
 
     def test_links_linkedin(self):
         dir_path = os.path.dirname(os.path.realpath(__file__))
-        f = open(dir_path + "/test_linkedin_links.txt").read()
+        mock_response = open(dir_path + "/test_linkedin_links.txt").read()
```
Comment (L1ghtn1ng) [inline tests/discovery/test_linkedin_links.py]: As you are using open() do not forget to tidy up after you are finished with the file with a .close()

**Comment by L1ghtn1ng** [comment]
Still have travis issues https://travis-ci.com/laramies/theHarvester/jobs/232109871

File: theHarvester/parsers/myparser.py [hunk 7]
```diff
@@ -84,7 +84,7 @@ def links_linkedin(self):
         for x in self.temp:
```
Comment (L1ghtn1ng) [inline theHarvester/parsers/myparser.py]: Can you update this single var and the one below please

**Comment by L1ghtn1ng** [comment]
Still failing with travis https://travis-ci.com/laramies/theHarvester/jobs/232111337

**Comment by L1ghtn1ng** [comment]
Still failing https://travis-ci.com/laramies/theHarvester/jobs/232112020 you can check to make sure the test passes by keeping an eye on https://travis-ci.com/laramies/theHarvester/pull_requests

**Comment by L1ghtn1ng** [comment]
What are you trying assert on that it is a list or not none etc?

**Comment by chrissparksnj** [comment]
> What are you trying assert on that it is a list or not none etc?

just getting the hang of what im actually doing.  i was trying to assert that a list is returned. but i run pytest locally and it says everything is fine.

**Comment by L1ghtn1ng** [comment]
Try
assert links == [] and see what happens
________________________________

**Comment by L1ghtn1ng** [comment]
Thanks for this

**Pull request merged**